error-prone-contrib/pom.xml

```diff
@@ -177,7 +177,7 @@
         <dependency>
             <groupId>org.mongodb</groupId>
             <artifactId>mongodb-driver-core</artifactId>
-            <scope>test</scope>
+            <scope>provided</scope>
```

Comment:
(Unrelated to this line)

Do we want MongoDB-related Refaster rules in the first place?